plaso/cli/helpers/elastic_output.py

```diff
@@ -60,6 +60,12 @@ def AddArguments(cls, argument_group):
         default=cls._DEFAULT_RAW_FIELDS, help=(
             'Export string fields that will not be analyzed by Lucene.'))
 
+    argument_group.add_argument(
```

Comment:
Can you change extra to additional to match dynamic output module terminology
https://github.com/log2timeline/plaso/blob/master/plaso/cli/helpers/dynamic_output.py#L40

Reply:
done